docs/howtos/DataPlane Generation - DPG/10longRunningOperations.mdx

```diff
@@ -0,0 +1,188 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Long-Running Operations
+
+This doc details what emitters will generate for long-running operations.
+
+## Resource create or replace
+
+Long-running operation to create or replace a resource.
+
+<Tabs>
+<TabItem value="typespec" label="TypeSpec" default>
+
+```typespec
+alias ResourceOperations = global.Azure.Core.ResourceOperations<NoConditionalRequests &
+  NoRepeatableRequests &
+  NoClientRequestId>;
+
+@resource("users")
+@doc("Details about a user.")
+model User {
+  @key
+  @visibility("read")
+  @doc("The name of user.")
+  name: string;
```

Comment:
Is it possible we make this name as optional not required? Since User would be used both input and output model, and the splitting model introducing by visibility is still under discussion. JS may not handle this required read-only yet.

Reply:
guide https://azure.github.io/typespec-azure/docs/getstarted/azure-core/step04

```
@doc("A widget.")
@resource("widgets")
model Widget {
  @key("widgetName")
  @doc("The widget name.")
  @visibility("read")
  name: string;

  @doc("The ID of the widget's manufacturer.")
  manufacturerId: string;
}
```

Reply:
If you have strong opinion, maybe we can see if we remove `@visibility("read")` and add `@path`

Reply:
Got it, no strong opinion here since it is also in typespec guide i think this is more js issue.